### PR TITLE
chore: No content layer

### PR DESCRIPTION
**Pull request by perkinsjr**
## What does this PR do?
Removes content layer because it's unmaintained and replaces it with remote-mdx which is maintained. 

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [Contributing Guide](./CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand areas
- [ ] Ran `pnpm build`
- [ ] Ran `pnpm fmt`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

### CODE REVIEW

File: apps/web/components/landing/mdx-content.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"use client"; // This is required!
```
Comment (chronark) [inline apps/web/components/landing/mdx-content.tsx:1]: why?

File: apps/web/components/landing/mdx-content.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import { MDXRemote, type MDXRemoteSerializeResult } from "next-mdx-remote";
+
+type MdxContentProps = {
+  source: MDXRemoteSerializeResult;
+};
+
+/** Custom components here!*/
+const MdxComponents = {};
```
Comment (chronark) [inline apps/web/components/landing/mdx-content.tsx:10]: so here we put our H1, Table etc?

File: apps/web/content/blog/authors.ts [hunk 3]
```diff
@@ -7,7 +7,10 @@ export type Author = {
   };
 };
 
-export const authors = {
```
Comment (chronark) [inline apps/web/content/blog/authors.ts:10]: I don't mind, but why?

**Review by chronark — APPROVED** [review]

File: apps/web/content/blog/authors.ts (hunk 3 above)
Comment (perkinsjr) [reply apps/web/content/blog/authors.ts:10]: This satisfied types in external files when using authors[string] when using satisfied it complains

File: apps/web/components/landing/mdx-content.tsx (hunk 1 above)
Comment (perkinsjr) [reply apps/web/components/landing/mdx-content.tsx:1]: There is a useState used to display the serialized data.

**Review by chronark — APPROVED** [review]

**Review by chronark — APPROVED** [review]

**Pull request merged**